[0m21:58:23.393342 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ba7e85c1550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ba7e9ee2210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ba7e7c4dd10>]}


============================== 21:58:23.395865 | 0355ab8a-8e5d-4094-8b9e-66f8d29ef818 ==============================
[0m21:58:23.395865 [info ] [MainThread]: Running with dbt=1.9.4
[0m21:58:23.396205 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/home/bd/.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init dbt_duckdb_postgres_demo', 'send_anonymous_usage_stats': 'True'}
[0m21:58:23.401798 [debug] [MainThread]: Starter project path: /home/bd/code/quarto/blog2310/posts/dbt-duckdb-postgres-tutorial/.venv/lib/python3.13/site-packages/dbt/include/starter_project
[0m21:58:23.402901 [info ] [MainThread]: 
Your new dbt project "dbt_duckdb_postgres_demo" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m21:58:23.403125 [info ] [MainThread]: Setting up your profile.
[0m21:58:25.435311 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 2.074387, "process_in_blocks": "4800", "process_kernel_time": 0.061294, "process_mem_max_rss": "118256", "process_out_blocks": "72", "process_user_time": 0.798521}
[0m21:58:25.435683 [debug] [MainThread]: Command `dbt init` succeeded at 21:58:25.435612 after 2.07 seconds
[0m21:58:25.435945 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ba7e77e8050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ba7e3942570>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ba7e77cdd00>]}
[0m21:58:25.436210 [debug] [MainThread]: Flushing usage events
[0m21:58:26.215763 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m08:39:05.502981 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x78c2195a5550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x78c21af1e210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x78c218c35d10>]}


============================== 08:39:05.506015 | 03f1b931-e942-4f79-9ed4-7a22aae9a196 ==============================
[0m08:39:05.506015 [info ] [MainThread]: Running with dbt=1.9.4
[0m08:39:05.506368 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/bd/.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m08:39:05.506656 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/bd/code/quarto/blog2310/posts/dbt-duckdb-postgres-tutorial/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m08:39:05.507351 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": false, "command_wall_clock_time": 0.037006944, "process_in_blocks": "61280", "process_kernel_time": 0.082799, "process_mem_max_rss": "97316", "process_out_blocks": "16", "process_user_time": 0.777461}
[0m08:39:05.507639 [debug] [MainThread]: Command `dbt run` failed at 08:39:05.507574 after 0.04 seconds
[0m08:39:05.507878 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x78c218a1ac40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x78c218839d90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x78c2189c5f20>]}
[0m08:39:05.508148 [debug] [MainThread]: Flushing usage events
[0m08:39:06.116919 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m17:41:46.854176 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x707d2d1c5550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x707d2eaca210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x707d2c859d10>]}


============================== 17:41:46.856303 | 80d76c4c-d4cd-4ee8-9f8e-1d466066dbf7 ==============================
[0m17:41:46.856303 [info ] [MainThread]: Running with dbt=1.9.4
[0m17:41:46.856634 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/bd/.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt docs generate', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m17:41:46.856918 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/bd/code/quarto/blog2310/posts/dbt-duckdb-postgres-tutorial/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m17:41:46.857515 [debug] [MainThread]: Resource report: {"command_name": "generate", "command_success": false, "command_wall_clock_time": 0.036056954, "process_in_blocks": "0", "process_kernel_time": 0.055693, "process_mem_max_rss": "97640", "process_out_blocks": "8", "process_user_time": 0.767669}
[0m17:41:46.857802 [debug] [MainThread]: Command `dbt docs generate` failed at 17:41:46.857741 after 0.04 seconds
[0m17:41:46.858022 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x707d2c442d70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x707d2c459d90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x707d2c5e5f20>]}
[0m17:41:46.858268 [debug] [MainThread]: Flushing usage events
[0m17:41:47.665475 [debug] [MainThread]: An error was encountered while trying to flush usage events
